RELEASES.md

```diff
@@ -96,38 +96,59 @@ deadline of 3 weeks.
 
 ### Cutting a major release
 
+#### In the week prior to the release
+
+* Check the [release milestone](https://github.com/envoyproxy/envoy/milestones).
 * Take a look at open issues tagged with the current release, by
   [searching](https://github.com/envoyproxy/envoy/issues) for
   "is:open is:issue milestone:[current milestone]" and either hold off until
   they are fixed or bump them to the next milestone.
-* Begin marshalling the ongoing PR flow in this repo. Ask maintainers to hold off merging any
-  particularly risky PRs until after the release is tagged. This is because we aim for main to be
-  at release candidate quality at all times.
+* Begin marshalling the ongoing PR flow in this repo.
+* Ask maintainers to hold off merging any particularly risky PRs until after the release is tagged.
+* Conversely, encourage maintainers to land anything that should be included.
 * Do a final check of the [release notes](changelogs/current.yaml):
```

Comment:
Consider moving it to the "day before" section.

Reply:
im thinking we should look over these with a bit of time as it takes time to do and creates changelog conflict so its good to get this out of the way with a bit of time